### PR TITLE
port: forth

### PR DESCRIPTION
**Pull request by glennj**
port: forth

This is a tricky exercise

[canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/grep/canonical-data.json)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/forth/example.sh [hunk 1]
```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+set -f                  # disable path exansion
+shopt -s extglob        # enable extended patterns
+
+# I'm going to use the stack and the macros as 
+# global variables: Not best practice.
```
Comment (kotp) [inline exercises/forth/example.sh]: That's fine, this is just a sanity test solution, doesn't have to be exemplary.

File: config.json [hunk 2]
```diff
@@ -390,6 +390,20 @@
         "text_formatting",
         "control_flow_loops"
       ]
+    },
+    {
+      "slug": "forth",
+      "uuid": "55c0913c-8d85-4330-868c-bca174d05d87",
```
Comment (guygastineau) [inline config.json]: This PR give the same exercise uuid to `forth` that is in use for `beer-song`:
`55c0913c-8d85-4330-868c-bca174d05d87`

Please use configlet to generate a new unique uuid for all ports.  Copying uuids from other tracks will result in collisions like the CI failure for this commit indicates.

Comment (glennj) [reply config.json]: done

Comment (guygastineau) [reply config.json]: Thank you.

**Comment by guygastineau** [comment]
Very nice work.

**Comment by guygastineau** [comment]
@glennj

A mentored left the following comment about the forth exercise on our slack.

'The Forth problem is a fun/interesting challenge
Kudos to whomever put that together'

I thought you deserved to have the praise forwarded to you.

**Pull request merged**